sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -1181,10 +1180,25 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
 
   test("current_database with multiple sessions") {
     sql("create database a")
+    // TestHiveContex
+    // scalastyle:off println
+    println("location of database a:" + hiveCatalog.getDatabase("a").locationUri)
+    println("list of databases:" + hiveCatalog.listDatabases())
+    // scalastyle:on println
+    sql("select current_database()").show()
     sql("use a")
+    sql("select current_database()").show()
```

Comment:
Based on the latest log: https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/55456/consoleFull, 
`sql("use a")` does not work. No error message is issued in the log. However, the following `s2.sql("use b")` works.